tests/unit/payments.js

```diff
@@ -165,6 +155,24 @@ define('tests/unit/payments',
             });
         });
 
+        it('defines a configurable adapter', function(done) {
+            var adapterConfigured = false;
+
+            setupMocks().mock('fxpay', {
+                init: function(opt) {
+                    // This simulates how fxpay will configure
+                    // the Marketplace adapter when fxpay.init()
+                    // is called.
+                    opt.adapter.configure({});
```

Comment:
mmm, yeah, sinon doesn't buy much in this case since the adapter instance doesn't exist until init(), as you mention.